ocl/OCL.g4

```diff
@@ -19,9 +19,9 @@ specification
   : 'package' ID '{' classifier* '}' EOF
   ;
 
-expressions
-  : expression (';' expression)* ';'? EOF
-  ;
+// expressions
+//   : expression (';' expression)* ';'? EOF
+//   ;
```

Comment:
Please remove not used code.

Reply:
Either the `expressions` rule be uncommented so the tests pass, or the rule, the test examples-expr/expressionExamples.txt, and the corresponding test in desc.xml all removed. It has to be one way other the other.

Reply:
Yes, uncomment the "expressions" rule.